content/lxd/getting-started-cli.md

```diff
@@ -78,17 +78,17 @@ To install the feature branch of LXD, run:
 
 #### Ubuntu
 ##### Ubuntu (all releases)
-The recommended way to install LXD is with the snap.
+The recommended way to install LXD is the [snap package](https://snapcraft.io/lxd).
 
-For the latest stable feature release, use:
+For the feature release, use:
```

Comment:
I would keep "latest" (or "current") here, since there are other feature releases, but the one that is installed is the latest.

Getting rid of "stable" makes sense for this guide though.

Reply:
Yes agreed with regards to keeping the word "latest" - for the reason you specify *and* because it aligns with the snap channel you will be installing from using that command which will be equivalent to running:

```
sudo snap install lxd --channel=latest/stable
```

However with regards to getting rid of the word "stable", I am not so sure. You can see in the equivalent command above that it does make sense to call it the "latest stable feature release".

This is because the snap channel "latest/stable" won't just contain each month's tagged feature release (4.16, 4.17 etc) it will also include rolling bug fixes throughout the time between the monthly tagged releases (where they are deemed important enough).

When your system gets the tagged feature release as well as the intermediate updates depends on what sub-channel you are on.

If you are on `latest/candidate` channel rather than `latest/stable` your system will receive the tagged release and any intermediate update before those on the `latest/stable` channel, and those on the `latest/edge` channel will get the updates even sooner (before they are tagged in fact).

You can see a list of them in the drop down top right on this page: https://snapcraft.io/lxd

Equally on the LTS channels, there are 3 sub channels `4.0/stable`, `4.0/candidate` and `4.0/edge` - with the same behaviour as `latest/*` channels but pulling from the stable-4.0 branch (https://github.com/lxc/lxd/tree/stable-4.0).

To complicate things even further there are also dedicated specific feature release channels, which allow users to pin their systems to a specific feature release. These are called `4.{n}/stable` and `4.{n}/candidate`.

So "latest stable feature release" is correct. 

Perhaps a brief introduction would be useful just before this section advising to the user which channel to pick (i.e the different support durations and release cycles).

For more info see https://discuss.linuxcontainers.org/t/managing-the-lxd-snap/8178

Reply:
A new user will generally want to use latest/stable though and doesn't really care about candidate or edge.
Adding more info here will make it harder to follow the steps. How about just linking to the tutorial instead?

Reply:
Yes agreed. I don't believe we should go into explaining all the different channel types (that extended explanation above was there to explain the wider channel concept), but to indicate why it was important to keep the words "latest stable", as well as defining (briefly) the difference between "latest stable feature" and "latest stable LTS" - they are both *stable* releases, but the concept of stability differ. :)

Reply:
> A new user will generally want to use latest/stable though

Thats an interesting point too, I'm not sure whether LTS or Feature channels are most appropriate for new users (in terms of just getting started without doing too much reading).

When you install Ubuntu and get LXD installed automatically it comes from the `4.0/stable` LTS channel, as that is an LTS release.

Arguably if you're an entirely new user, you'd be better of on the 4.0 LTS release as it will have less changes occurring, and if you do want to upgrade to the feature release channel to get access to the new features you can do that, whereas if you decide you want to have the LTS level of stability, if you're already on the feature release channel you cannot switch to the LTS channel.

So maybe it is safer to encourage users to start with the LTS and switch to feature if they need it?

 I'm sure @stgraber will have a view on that though.

Reply:
Reading through this, I agree that we should keep saying `latest`. For those not familiar with snaps, this will give them an indication that they don't need to do anything to get a specific release.

As for 4.0 vs latest, my preference for new user is to give them latest. As a new user, they're likely to go through our documentation, tutorials, release announcements, ... and so having some random features they read about be missing would be a bad experience. Then once they're ready to go to production and care a lot more about maintenance time and support length, that's when they'll learn about the LTS releases.

Reply:
So let's go for "the latest feature release" and add a sentence like "For information about other available versions, see [Managing the LXD snap](https://discuss.linuxcontainers.org/t/managing-the-lxd-snap/8178)." at the end of this section?

Reply:
Ok, I will readd the `latest` part.

The article about managing LXD snap is also interesting, but it sounds kind of complicated...I don't know whether you really want this for new users.
Maybe we could add this to the advanced guide instead :thinking:.

Reply:
The advanced guide starts after the installation though, and that is where you need this information. I think having a link is fine - users who don't care about other versions can ignore it.

Reply:
Ok. I will add the link as well.